### PR TITLE
Added an element for setting high_rep policy for unittests.

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

File: unittests/pom.xml [hunk 1]
```diff
@@ -108,14 +108,11 @@
...
-                    <!-- address>0.0.0.0</address>
-                    <port>8080</port -->
-                    <!-- Comment in the below snippet to enable local debugging with a remove debugger
-                         like those included with Eclipse or IntelliJ -->
-                    <!-- jvmFlags>
-                      <jvmFlag>-agentlib:jdwp=transport=dt_socket,address=8000,server=y,suspend=n</jvmFlag>
-                    </jvmFlags -->
+                    <!-- [START maven_test_highrep_configuration] -->
+                    <jvmFlags>
+                        <jvmFlag>-Ddatastore.default_high_rep_job_policy_unapplied_job_pct=20</jvmFlag>
```
Comment (lesv) [inline unittests/pom.xml]: Shouldn't we point to the docs where this comes from?
https://cloud.google.com/appengine/docs/java/tools/devserver#High_Replication_Consistency_Model

Comment (tmatsuo) [inline unittests/pom.xml]: Ok, done

**Comment by tmatsuo** [comment]
@lesv OK, added a link to the docs.

**Comment by lesv** [comment]
LGTM - it's warning me to merge w/ caution, so I'll leave that to you.

**Comment by tmatsuo** [comment]
well, on the second thought, this change is not appropriate, so I'm just closing.

**Pull request closed without merging**